src/components/Router/index.js

```diff
@@ -131,8 +130,8 @@ class CrossDeviceMobileRouter extends Component {
   }
 
   render = (props) =>
-    this.state.loading ? <WrappedSpinner i18n={this.state.i18n} disableNavigation={true} /> :
-      this.state.crossDeviceError ? <WrappedError i18n={this.state.i18n} disableNavigation={true} /> :
+    this.state.loading ? <WrappedSpinner disableNavigation={true} /> :
+      this.state.crossDeviceError ? <WrappedError disableNavigation={true} /> :
         <HistoryRouter {...props} {...this.state}
```

Comment:
Don't we need to wrap this component in `LocaleProvider`?

Reply:
Yes, we do. Thanks 😄